src/opower/utilities/oru.py

```diff
@@ -0,0 +1,26 @@
+"""Orange & Rockland Utilities (ORU)."""
+
+from typing import Optional
+
+import aiohttp
+from pyotp import TOTP
+
+from ..const import USER_AGENT
+from ..exceptions import InvalidAuth
+from .base import UtilityBase
+from .coned import ConEd
+
+HOSTNAME = "oru.com"

```

Comment:
I don't see how this overrides the one in coned.py. Have you tested your latest revision still works? If it does then I guess you can login to coned.com with oru.com credentials in which case you don't need any changes in coned.py and just go with your changes below. If it doesn't work, you have to extract the hostname to a static method and override it below.

Reply:
You will have to change only in coned.py:

from

```python
    @staticmethod
    async def async_login(
        session: aiohttp.ClientSession,
```
to:

```python
    @classmethod
    async def async_login(
        cls,
        session: aiohttp.ClientSession,
```

and then inside async_login you will be able to reference the @staticmethod that returns the hostname with: `cls.hostname()`

This also means some of the constants in coned.py will have to become variables in async_login.